meson/boost/meson.build

```diff
@@ -3,6 +3,7 @@ dep_boost = dependency('boost', version: '>= 1.42', required: true)
 # with C++11.
 have_boost_1_48_0 = dep_boost.version().version_compare('>= 1.48.0')
 conf.set('HAVE_BOOST_GE_148', have_boost_1_48_0, description: 'Boost version >= 1.48.0')
+conf.set('HAVE_BOOST_CONTAINER_FLAT_SET_HPP', have_boost_1_48_0, description: 'Boost version >= 1.48.0 has boost::container::flat_set')
```

Comment:
This seems to be recursor-only, so perhaps keeping it in the recursor top-level meson file makes more sense.